src/components/ProjectDashboard/components/NftRewardsPanel/NftReward/NftReward.tsx

```diff
@@ -0,0 +1,89 @@
+import { t } from '@lingui/macro'
+import { NftPreview } from 'components/NftRewards/NftPreview'
+import { PayProjectFormContext } from 'components/Project/PayProjectForm/payProjectFormContext'
+import { DEFAULT_NFT_MAX_SUPPLY } from 'contexts/NftRewards/NftRewards'
+import { NftRewardTier } from 'models/nftRewards'
+import { useContext, useState } from 'react'
+import { classNames } from 'utils/classNames'
+import { ipfsUriToGatewayUrl } from 'utils/ipfs'
+import { AddNftButton } from './AddNftButton'
+import { NftDetails } from './NftDetails'
+import { NftThumbnail } from './NftThumbnail'
+
+type NftRewardProps = {
+  rewardTier?: NftRewardTier
+  loading?: boolean
+  onSelect: (quantity?: number) => void
+  previewDisabled?: boolean
+  hideAttributes?: boolean
+}
+
+export function NftReward({
+  loading,
+  rewardTier,
+  previewDisabled,
+  onSelect,
+  hideAttributes,
+}: NftRewardProps) {
+  const { form: payProjectForm } = useContext(PayProjectFormContext)
```

Comment:
@johnnyd-eth , I think its all good to merge with this stuff as it is. I can fix it up and re-route to the project cart 👍

Reply:
@wraeth-eth beautiful sir sounds good 👍